### PR TITLE
Non udemy tests feature branch

### PR DESCRIPTION
**Pull request by matheszandras**

### CODE REVIEW

File: src/test/java/practice/pageobject/CookieSettingsObjects.java [hunk 1]
```diff
@@ -1,36 +1,58 @@
...
-        super(driver);
+    public CookieSettingsObjects() {
+        super();
     }
 
     @FindBy(className = "ot-sdk-container")
-    public WebElement cookiePopup;
+    private WebElement cookiePopup;
     @FindBy(className = "cookie-setting-link")
     public WebElement cookiePopupSettings;
```
Comment (ghost) [inline src/test/java/practice/pageobject/CookieSettingsObjects.java]: General comment: variables that are only used in the containing class should be private

File: src/test/java/practice/pageobject/NHLObjects.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    public void searchForJim(String name) {
+        searchIcon.click();
+        searchBar.sendKeys(name);
+        searchBar.sendKeys(Keys.ENTER);
+    }
+
+    public void latestNews() {
+        WebDriverWait wait = new WebDriverWait(Hooks.getWebDriver(), 3);
```
Comment (ghost) [inline src/test/java/practice/pageobject/NHLObjects.java:52]: For future reference: let's come up with a better approach for these waits in the next framework

File: target/test-classes/practice/hooks/Hooks.class
Comment (ghost) [inline target/test-classes/practice/hooks/Hooks.class:1]: Please never commit binaries to source code repositories

File: target/test-classes/practice/stepdefs/CommonStepDefs.class
Comment (ghost) [inline target/test-classes/practice/stepdefs/CommonStepDefs.class:1]: Please never commit binaries to source code repositories

**Review by ghost — APPROVED** [review]

File: target/test-classes/practice/hooks/Hooks.class
Comment (matheszandras) [reply target/test-classes/practice/hooks/Hooks.class:1]: Fixed

File: target/test-classes/practice/stepdefs/CommonStepDefs.class
Comment (matheszandras) [reply target/test-classes/practice/stepdefs/CommonStepDefs.class:1]: Fixed

File: src/test/java/practice/pageobject/CookieSettingsObjects.java (hunk 1 above)
Comment (matheszandras) [reply src/test/java/practice/pageobject/CookieSettingsObjects.java]: I forgot it completely... Fixed now!

File: src/test/java/practice/pageobject/NHLObjects.java (hunk 2 above)
Comment (matheszandras) [reply src/test/java/practice/pageobject/NHLObjects.java:52]: OK. Noted.

**Pull request merged**